src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -123,6 +123,7 @@ def load_arguments(self, _):
         c.argument('appgw_subnet_id', options_list=['--appgw-subnet-id'], arg_group='Application Gateway')
         c.argument('appgw_watch_namespace', options_list=['--appgw-watch-namespace'], arg_group='Application Gateway')
         c.argument('aci_subnet_name', type=str)
+        c.argument('enable_encryption_at_host', options_list=['--enable-encryption-at-host'], action='store_true')
```

Comment:
Is this a property that will be persisted in resource? If so, then `get_three_state_flag()` is recommended instead of using `action='store_true'`

Reply:
hi @fengzhou-msft what's the diff between `store_true` and `get_three_state_flag`? I found all AKS code are using `store_true`:
```
        c.argument('enable_cluster_autoscaler', options_list=["--enable-cluster-autoscaler", "-e"], action='store_true')
        c.argument('disable_cluster_autoscaler', options_list=["--disable-cluster-autoscaler", "-d"], action='store_true')
        c.argument('update_cluster_autoscaler', options_list=["--update-cluster-autoscaler", "-u"], action='store_true')
```

Do you mean I should change to following? Thanks.
```
c.argument('enable_encryption_at_host', arg_type=get_three_state_flag(), help='Enable EncryptionAtHost.'
```

Reply:
thanks, fixed.